general/documentation/contributing.md

```diff
@@ -31,6 +33,77 @@ Anyone can get involved with, and add to, these resources - you don't have to be
 
 If you need help, please reach out to us and we will do our best to advise you.
 
+### Migrating legacy docs
+
+We are currently migrating documentation from our [legacy documentation site](https://docs.moodle.org/dev), and one of the best ways to contribute with documentation is to help with this migration process.
+
+We have created a number of tools to make this a little easier and to automate as much as possible. These are mostly focused around our migration assistant. You can find its full documentation by running:
+
+```console
+yarn migrate --help
+```
+
+#### What we automate
+
+The migration assistant will:
+
+- try to guess the:
+  - page title
+  - relevant tags
+- create the directory structure to the new markdown file
+- fetch any images included in the original page
+- convert:
+  - WikiText tables to either Markdown tables, or HTML tables as appropriate
+  - page headings
+  - lists
+  - common font formatting like bold, and italics
+  - code blocks
+  - some common `*Note*` syntaxes to use an [Admonition](https://docusaurus.io/docs/next/markdown-features/admonitions)
+  - InterWiki links to fully-qualified links
+  - Regular links to markdown links
+- run markdownlint in fix mode to automatically fix any common, fixable issues
+- update the links for any internal documentation which has already been migrated
+- add the migrated page to our [list of migrated pages](https://github.com/moodle/devdocs/main/blob/data/migratedPages.yml)
+- update any links to the migrated page
+
+#### Using the assistant
+
+In most circumstances you should not need to pass any special options, and you will just need to pass the name of the document you are migrating, and the path to the location you'd like to migrate it to, for example:
+
+```console title="Migrate the 'Moodle and PHP' documentation to a new policies page"
```

Comment:
This title should be removed or replaced with something more generic (the Moodle and PHP one is below).